src/components/Table/columns/BooleanColumn.tsx

```diff
@@ -15,14 +16,7 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-const BooleanColumn = ({
-    result,
-    field,
-    path,
-    variant = "outlined",
-    size = "small",
-    color = "default",
-}: TableColumnPathType & ChipProps) => {
+const BooleanColumn = ({ result, field, path, chipProps }: TableColumnPathType & ChipFieldInterface) => {
```

Comment:
Please also export TableColumnPathType & ChipFieldInterface as a BooleanColumnProps at the bottom.

i.e.:
```ts
type BooleanColumnProps = TableColumnPathType & ChipFieldInterface;
```

and then use it and export